pages/create/[slug]/index.js

```diff
@@ -15,14 +17,22 @@ export default function EmotionEntry({
   }, []);
 
   const router = useRouter();
+  const { id } = router.query;
+  const { slug } = router.query;
+
+  const { data: correspondingEntryFromMongo, isLoading } = useSWR(
```

Comment:
Also here I personally would prefer another naming like correspondingDbEmotionEntry, but that's of course a matter of taste.